packages/common/src/extensionDependencies.ts

```diff
@@ -5,7 +5,7 @@ export const extensionDependencies = [
   "scalameta.metals",
   "mrob95.vscode-talonscript",
   "jrieken.vscode-tree-sitter-query",
-
   // Necessary for the `drink cell` and `pour cell` tests
   "ms-toolsai.jupyter",
+  "jakebecker.elixir-ls",
```

Comment:
after you update this one to latest main, you'll want to move this extension a couple lines up and add a comment